la.html

```diff
@@ -0,0 +1,42 @@
+<!doctype html>
+<html>
+<head>
+<title>Our Funky HTML Page</title>
+<link rel="stylesheet" type="text/css" href="./vlog.css"/>
+<meta name="description" content="Our first page">
+<meta name="keywords" content="html tutorial template">
+</head>
+<body>
+
+<div class="main-menu">
+    <div class="main-menu_top">
+        <div class="logo"> 
+            <img class="big_logo" src="linklogo.png">
```

Comment:
if it's a modification I'd recommend using logo_big, see [BEM](https://en.bem.info/methodology/naming-convention/)

Reply:
Done